patches/sitecustomize.py

```diff
@@ -26,6 +33,10 @@ def monkeypatch_bq(bq_client, *args, **kwargs):
                 if (not has_bigquery):
                     print('Please ensure you have selected a BigQuery '
                           'account in the Kernels Settings sidebar.')
+            print(kwargs)
```

Comment:
You probably want to remove these because I guess these were for debugging.

Reply:
Thanks for catching that!